README.md

```diff
@@ -6,13 +6,14 @@ This project is part of the [Spring organization][] on GitHub.
+|--------------------------------------------------------------------|-------------------------------|------------------------------------------------------|----------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [spring-batch-bigquery](spring-batch-bigquery/README.adoc)         | Support for [Google BigQuery] | [@dgray16](https://github.com/dgray16)               | 0.1.0          | [![Spring Batch BigQuery](https://github.com/spring-projects/spring-batch-extensions/actions/workflows/spring-batch-bigquery.yml/badge.svg)](https://github.com/spring-projects/spring-batch-extensions/actions/workflows/spring-batch-bigquery.yml)                           |
+| [spring-batch-elasticsearch](spring-batch-elasticsearch/README.md) | Support for [Elasticsearch]   | [@parikshitdutta](https://github.com/parikshitdutta) | 0.1.0-SNAPSHOT | [![Spring Batch Elasticsearch](https://github.com/spring-projects/spring-batch-extensions/actions/workflows/spring-batch-elasticsearch.yml/badge.svg)](https://github.com/spring-projects/spring-batch-extensions/actions/workflows/spring-batch-elasticsearch.yml)            |
+| [spring-batch-excel](spring-batch-excel/README.adoc)               | Support for [Microsoft Excel] | [@mdeinum](https://github.com/mdeinum)               | 0.1.1          | [![Spring Batch Excel](https://github.com/spring-projects/spring-batch-extensions/actions/workflows/spring-batch-excel.yml/badge.svg)](https://github.com/spring-projects/spring-batch-extensions/actions/workflows/spring-batch-excel.yml)                                    |
+| [spring-batch-geode](spring-batch-geode/README.adoc)               | Support for [Apache Geode]    | TBA                                                  | 0.1.0-SNAPSHOT | [![Spring Batch Geode](https://github.com/spring-projects/spring-batch-extensions/actions/workflows/spring-batch-geode.yml/badge.svg)](https://github.com/spring-projects/spring-batch-extensions/actions/workflows/spring-batch-geode.yml)                                    |
+| [spring-batch-neo4j](spring-batch-neo4j/README.md)                 | Support for [Neo4j]           | [@michael-simons](https://github.com/michael-simons) | 0.1.0          | [![Spring Batch Neo4j](https://github.com/spring-projects/spring-batch-extensions/actions/workflows/spring-batch-neo4j.yml/badge.svg)](https://github.com/spring-projects/spring-batch-extensions/actions/workflows/spring-batch-neo4j.yml)                                    |
```

Comment:
I thought it would be nice to see the modules appear alphabetically and directly linked to their README.

I also reordered the links at the bottom, although this does not impact the final rendering.

If you don't like the proposal, let me know and I'll revert the changes 🙂

Reply:
Great! This change is welcome as well 👍